evm/README.md

```diff
@@ -1,67 +1,172 @@
-## Foundry
+# Message Lifecycle (EVM)
+
+1. **Send**
+
+A client calls on [`transfer`] to initiate an NTT transfer. The client must specify at minimum, the amount of the transfer, the recipient chain, and the recipient address on the recipient chain. [`transfer`] also supports a flag to specify whether the NttManager should queue transfers if they are rate-limited (see [link to Rate Limiting Section]). Clients can also include additional instructions to forward along to its peer NTT Manager on the recipient chain to execute. Depending on mode set in the initial configuration of the NttManager contract, transfers are either "locked" or "burned". Once the transfer has been forwarded to the Transceiver, the NttManager emits the `TransferSent` event.
```

Comment:
```suggestion
A client calls [`transfer`] to initiate an NTT transfer. The client must specify, at minimum, the amount of the transfer, the recipient chain, and the recipient address on the recipient chain. [`transfer`] also supports a flag to specify whether the NttManager should queue or revert transfers if they are rate-limited (see [link to Rate Limiting Section]). Depending on mode set in the initial configuration of the NttManager contract, transfers are either "locked" or "burned". Once the transfer has been forwarded to the Transceiver, the NttManager emits the `TransferSent` event.
```
there's no additional information that goes to the ntt manager peer, right?

Reply:
yea its to the Transceiver - will update